src/screens/internal/ShareImage/ShareCardImage.tsx

```diff
@@ -50,21 +44,24 @@ const Header = (props: { isHomePage?: Boolean }) => {
   );
 };
 
-const ShareCard = ({ stateId, countyFipsId }: ShareCardProps) => {
-  if (stateId || countyFipsId) {
-    return <LocationShareCard stateId={stateId} countyFipsId={countyFipsId} />;
+interface ShareCardProps {
+  region: Region | null;
+}
+
+const ShareCard = ({ region }: ShareCardProps) => {
+  if (region) {
+    return <LocationShareCard region={region} />;
   } else {
     return <SocialLocationPreview />;
   }
 };
 
-const LocationShareCard = ({ stateId, countyFipsId }: ShareCardProps) => {
-  let projections: Projections | undefined;
-  const [countyOption] = useState(
-    countyFipsId && findCountyByFips(countyFipsId),
-  );
-  stateId = stateId || countyOption.state_code;
-  projections = useProjections(stateId!, countyOption) as any;
+interface LocationShareCardProps {
+  region: Region;
+}
+
+const LocationShareCard = ({ region }: LocationShareCardProps) => {
```

Comment:
Maybe this component should use a projection instead? It doesn't seem that the component uses the region for anything other than getting the projections. We can do that later

Reply:
Yeah I think ideally it would, probably will keep it as is for now